gpcontrib/arenadata_toolkit/expected/adb_vacuum_strategy_test.out

```diff
@@ -1,10 +1,13 @@
 CREATE EXTENSION arenadata_toolkit;
+-- Change log level to disable notice messages from PL/pgSQL
+SET client_min_messages=WARNING;
 SELECT arenadata_toolkit.adb_create_tables();
  adb_create_tables 
 -------------------
  
 (1 row)
 
+RESET client_min_messages;
```

Comment:
You decided to RESET here in order to leave vacuum tests untouched? (I see that vacuum tests have their own SET RESET pair).
Why in `adb_get_relfilenodes_test.sql` the whole test is just wrapped by SET at the begining and RESET at the end, but meanwhile  in `adb_collect_table_stats_test.sql`  `client_min_messages` is SET at the begining, and at the end it's not RESET? 
Maybe, if you've started to make the tests idependent and clear the enviroment, it would be also better to **unify** the style with warnings? Otherwise there is some inconsistency in style.